test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -540,6 +540,30 @@ def forward(self, x):
         x = torch.randn(2, 3, 4)
         self.run_test(ArithmeticModule(), x)
 
+    def test_floor_div(self):
+        class FloorDivModule(torch.nn.Module):
+            def forward(self, x, y):
+                return x // 3, x // 2., \
+                    x.to(dtype=torch.float64) // 3, x.to(dtype=torch.float64) // 2., \
+                    x.to(dtype=torch.int64) // 3, x.to(dtype=torch.int64) // 2., \
+                    x // (y + 1.).to(dtype=torch.int64), x // y, \
+                    x.to(dtype=torch.float64) // y.to(dtype=torch.int64), x.to(dtype=torch.float64) // y.to(dtype=torch.int64), \
```

Comment:
There's a duplicate  x.to(dtype=torch.float64) // y.to(dtype=torch.int64) here

Reply:
good catch!